changedetectionio/fetch_site_status.py

```diff
@@ -16,6 +17,28 @@ def __init__(self, *args, datastore, **kwargs):
         super().__init__(*args, **kwargs)
         self.datastore = datastore
 
+    def html_to_text(self, html_content: str) -> str:
```

Comment:
Move this to https://github.com/dgtlmoon/changedetection.io/blob/master/changedetectionio/html_tools.py